Network/centurylink-cloud-public-ip-listing.md

```diff
@@ -0,0 +1,124 @@
+{{{ "title": "CenturyLink Cloud Public IP Listing",
+"date": "07-06-2016",
+"author": [],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+
+CenturyLink Cloud customers may wish to add a public IP to specific [virtual machine](/servers/) or [Bare Metal](/bare-metal/) server in their cloud environment to deliver services. Public IPs are delivered using a 1 to 1 NAT model.
+
+### General Notes & Best Practices
+
+* All public IPs deployed on the platform have [hairpinning](/knowledge-base/network/hairpin-nats/) enabled.
```

Comment:
make sure you use relative paths as noted by Rich for specific KBs (see the readme)

```
[hairpinning](../Network/hairpin-nats.md)
```